packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -87,8 +87,15 @@ class Compiler {
         // eslint-disable-next-line  no-async-promise-executor
         return new Promise(async (resolve) => {
             await this.compiler.run((err, stats) => {
-                const content = this.compilerCallback(err, stats, lastHash, options, outputOptions);
-                resolve(content);
+                if (this.compiler.close) {
```

Comment:
are there instances when `.close` is not available?

Reply:
On versions other than v5